lib/api/kuzzle.js

```diff
@@ -192,7 +191,7 @@ function registerErrorHandlers(kuzzle) {
     // SIGSEGV: 11, can not be handled
     SIGPIPE: 13,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this trailing comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ATrailingComma)